src/ImageProcessor.Web/HttpModules/ImageProcessingModule.cs

```diff
@@ -202,7 +202,15 @@ public static void SetHeaders(HttpContext context, string responseType, string[]
 
             cache.SetExpires(DateTime.Now.ToUniversalTime().AddDays(maxDays));
             cache.SetMaxAge(new TimeSpan(maxDays, 0, 0, 0));
-            cache.SetRevalidation(HttpCacheRevalidation.AllCaches);
+
+            if (ParseCacheBuster(context.Request.QueryString))
+            {
+                cache.AppendCacheExtension("immutable");
+            }
+            else
+            {
+                cache.SetRevalidation(HttpCacheRevalidation.AllCaches);
```

Comment:
Any reason why we should require stale resources to be re-validated?

Reply:
Looking into it, this seems to be the default behavior, but makes it unambiguous:
- https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching#Cache_validation
- https://stackoverflow.com/a/8729854/96175

CloudVPS also recommends adding this to the `Cache-Control` header to make sure caches honor the `max-age` value: https://www.cloudvps.com/helpcenter/content-delivery-network-cdn/cdn-cache-control-and-invalidations.

So you'd still want to leave it in... I was just double checking 😃

Reply:
Ace, let's merge your PR then 👍